src/test/kotlin/com/atlassian/performance/tools/report/api/result/LocalRealResult.kt

```diff
@@ -24,14 +27,37 @@ class LocalRealResult(
     )
 
     fun loadRaw(): RawCohortResult = RawCohortResult.Factory().fullResult(
-        cohort = path.joinToString(separator = "/") { it.toString() },
-        results = File(
+        cohort = relativeResourcePath.joinToString(separator = "/") { it.toString() },
+        results = extractDirectory()
+    )
+
+    private fun extractDirectory(): Path {
+        val file = File(
             this::class
                 .java
-                .getResource("real-results/$path")
+                .getResource("real-results/$relativeResourcePath")
                 .toURI()
-        ).toPath()
-    )
+        )
+        return if (file.name.endsWith(".zip")) {
```

Comment:
What's going on here? I'm having troubles to understand? Commit message says;
> Assert on timeline content

Can we extract this `if` into a method?

Reply:
We're unzipping a big result for test purposes.

The full  b320be2584cd492e6e837976385aede213d71c5e message said:
> PS. Consider this a repro for [JPERF-395](https://ecosystem.atlassian.net/browse/JPERF-395).
This fails with OOME.
Raw data for 72 VUs over a 20 minute load weighs 617 MiB.
It weighs 46 MiB after zipping, which proves how much unnecessary repetition there is.

I extracted the `if` body. I also improved the commit message to explicitly mention the big test zip.